services/scan/src/interpreter.test.ts

```diff
@@ -299,9 +301,603 @@ test('interprets marks on an upside-down HMPB', async () => {
     ).interpretation as InterpretedHmpbPage
   ).toMatchInlineSnapshot(`
     Object {
-      "actualElectionHash": "602c9b551d08a348c3e1",
-      "expectedElectionHash": "965aa0b918b9bab9a2a4",
-      "type": "InvalidElectionHashPage",
```

Comment:
This seems like a snapshot that got changed inaccurately / accidentally in a past PR.